customers/tests/test_profile_service.py

```diff
@@ -0,0 +1,87 @@
+from unittest import mock
```

Comment:
Running of this test is giving:
`django.core.exceptions.ImproperlyConfigured: Set the PROFILE_API_URL environment variable`

Reply:
Have you added the variable to your `.env`?